### PR TITLE
Add task for running performance tests in Concourse

### PR DESCRIPTION
**Pull request by jcox-dev**
Adapted from acceptance tests repo.

### CODE REVIEW

File: tasks/kubectl-run-performance-tests.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - |
+      cat >~/gcloud-service-key.json <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+
+      # Use gcloud service account to configure kubectl
+      gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+      gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
+
+      SFTP_TARGET_DIRECTORY=$(kubectl get configmap project-config -o=jsonpath="{.data.sftp-target-directory}")
```
Comment (AdamHawtin) [inline tasks/kubectl-run-performance-tests.yml]: I think the variable is unused, `.data.sftp-target-directory` doesn't exist anymore in any case.

Comment (jcox-dev) [reply tasks/kubectl-run-performance-tests.yml]: @AdamHawtin probably needs removing from the acceptance tests task as well then.

**Comment by jcox-dev** [comment]
@AdamHawtin it currently uses the `350000_sample_file` (defined in the `kubernetes.env` file):

```bash
SAMPLE_FILE_PATH=resources/sample_files/350000_sample_file.csv
ACTION_RULE_DELAY_MINUTES=30
PRINT_FILE_TIME_LIMIT_MINUTES=30
```

**Comment by AdamHawtin** [comment]
@jcox-dev Makes sense, I guess there's no reason to run it in a pipeline with any other file at the moment.

**Pull request merged**